packages/ButtonLink/ButtonLink.jsx

```diff
@@ -87,6 +98,11 @@ ButtonLink.propTypes = {
    * Target URL
    */
   href: PropTypes.string,
+  /**
+   * If `true`, sets `ButtonLink` to 100% width.
+   * @since 2.2.0
+   */
+  fullWidth: PropTypes.oneOf(['true', 'false']),
```

Comment:
Can we use a boolean instead of true/false strings?

Reply:
Hi @jraff,
I have changed boolean instead of true/false